src/main/java/Storage.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+
+import java.io.IOException;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.FileNotFoundException;
+
+import java.nio.file.Paths;
+import java.nio.file.Files;
+
+/**
+ * Encapsulates serialization and external storage of the task list state.
+ */
+public class Storage {
+    public static final String STORAGE_SAVE_PATH = "data/duke.txt";
+    
+    public Storage() {
+    }
+    
+    @SuppressWarnings("unchecked")
```

Comment:
I think using SuppressWarnings works here but I believe it may not be a good coding practice to follow; perhaps you could consider changing some implementation to remove it? Once again, just a suggestion! :)